testing/btest/scripts/base/frameworks/logging/field-extension-cluster-error.zeek

```diff
@@ -12,6 +12,7 @@
 redef Cluster::nodes = {
 	["manager-1"] = [$node_type=Cluster::MANAGER, $ip=127.0.0.1, $p=to_port(getenv("BROKER_PORT1"))],
 	["worker-1"]  = [$node_type=Cluster::WORKER,  $ip=127.0.0.1, $p=to_port(getenv("BROKER_PORT2")), $manager="manager-1", $interface="eth0"],
+	["proxy-1"]   = [$node_type=Cluster::PROXY,   $ip=127.0.0.1, $p=to_port(getenv("BROKER_PORT2")), $manager="manager-1"],
```

Comment:
Does this fix a test that was already returning the error? Should there be a test that does so intentionally?

Reply:
Just seconding the idea that having a test that intentionally raises the error would be neat.